modules/Core/pages/panel/users_sessions.php

```diff
@@ -0,0 +1,101 @@
+<?php
+/*
+ *  Made by Supercrafter100
+ *  https://github.com/NamelessMC/Nameless/
+ *  NamelessMC version 2.0.3
+ *
+ *  License: MIT
+ *
+ *  Panel user sessions page
+ */
+if (!$user->handlePanelPageLoad('admincp.users.sessions')) {
+    require_once(ROOT_PATH . '/403.php');
+    die();
+}
+
+const PAGE = 'panel';
+const PARENT_PAGE = 'users';
+const PANEL_PAGE = 'sessions';
+$page_title = $language->get('admin', 'sessions');
+require_once(ROOT_PATH . '/core/templates/backend_init.php');
+
+// Load modules + template
+Module::loadPage($user, $pages, $cache, $smarty, [$navigation, $cc_nav, $staffcp_nav], $widgets, $template);
+
+if (!isset($_GET['id']) || !is_numeric($_GET['id'])) {
+    Redirect::to(URL::build('/panel/users'));
+}
+
+$view_user = new User($_GET['id']);
+if (!$view_user->exists()) {
+    Redirect::to(URL::build('/panel/users'));
+}
+
+if (isset($_GET['action'])) {
+    if ($_GET['action'] == 'logout' && isset($_GET['hash'])) {
+        $hash = $_GET['hash'];
+        DB::getInstance()->update('users_session', [
+            'hash',
+            '=',
```

Comment:
This formatting is a bit wonky, we can do this where array clause on 1 line (and you don’t need the “=“)